.github/workflows/k8s-tests.yml

```diff
@@ -55,19 +51,17 @@ on:  # yamllint disable-line rule:truthy
 jobs:
   tests-kubernetes:
     timeout-minutes: 240
-    name: "\
-      K8S System:${{ matrix.executor }} - ${{ matrix.use-standard-naming }} - \
-      ${{ inputs.kubernetes-versions-list-as-string }}"
+    name: "K8S System:${{ matrix.executor }}-${{ matrix.kubernetes-combo }}-${{ matrix.use-standard-naming }}"
     runs-on: ${{ fromJSON(inputs.runs-on-as-json-default) }}
     strategy:
       matrix:
         executor: [KubernetesExecutor, CeleryExecutor, LocalExecutor]
         use-standard-naming: [true, false]
+        kubernetes-combo: ${{ fromJSON(inputs.kubernetes-combos) }}
```

Comment:
We switch from running several k8s clusters in the same job - to run them "per job". They are heavy-enough to take the whole runner, and they are flaky-enough so that re-running them individually makes more sense.